.github/workflows/cli.yml

```diff
@@ -33,6 +29,11 @@ env:
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
Why was this added?

Reply:
Good question. I suspect I did so to avoid older builds taking up resources as new ones are started. It looks like the other workflows in this repository also have this enabled, which might be another contributing factor to me thinking it would be good to do the same here.

Any reason we don't want to do this?